var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -44,19 +44,26 @@ class Petsc(Package):
     version('3.5.1', 'a557e029711ebf425544e117ffa44d8f')
     version('3.4.4', '7edbc68aa6d8d6a3295dd5f6c2f6979d')
 
+    # If this has trouble finding BLAS and LAPACK...
```

Comment:
i think this comment should be close to `--with-lapack-lib` so that it's easier to read and related it to the code.